src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/Dumpling.targets

```diff
@@ -25,9 +25,16 @@
     <DumplingIncPaths Condition="'$(TargetOS)'=='OSX'" Include="/usr/lib/libSystem.B.dylib" />
     <DumplingIncPaths Condition="'$(TargetOS)'=='OSX'" Include="/usr/lib/libc++.1.dylib" />
   </ItemGroup>
+  <PropertyGroup Condition="'$(HELIX_PYTHONPATH)'==''">
```

Comment:
You can write this much simpler
```xml
  <PropertyGroup>
    <PythonPath>$(HELIX_PYTHONPATH)</PythonPath>
    <PythonPath Condition="'$(PythonPath)' == ''">python</PythonPath>
 </PropertyGroup>
```

Reply:
Also I think that would be idiomatic

Reply:
One of these days I should learn msbuild... :smile: currently I think I am writing more msbuild than c#...